workflows/phylorthology.nf

```diff
@@ -64,30 +66,12 @@ include { GENERAX                                   } from '../modules/local/gen
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { MAFFT                                 } from '../modules/nf-core/mafft/main' // Slightly modified (specific parameters)
-include { MAFFT as MAFFT_REMAINING              } from '../modules/nf-core/mafft/main'
 
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     RUN MAIN WORKFLOW
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
-// Function to get list of [ meta, [ file ] ]
```

Comment:
yeah, I'm glad you were able to get rid of this - felt a bit hacky/extraneous, and again, not having to deal with the tuples for all those downstream modules will be a big improvement.